ComicFormViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ComicFormViewModel.swift
```

Comment:
This file is currently living in the root of the repo - it should be moved to the right folder